src/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm.VscPreview
+{
+    using System;
+    using System.IO;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.Build.Engine;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                while (true)
+                {
+                    // get the path;
+                    string path = Console.ReadLine();                   // path -> basedir
+                    if (path == "exit")
+                        break;
+                    string filename = Console.ReadLine();               // filename -> the relative path of the current file 
```

Comment:
agree !

---
In reply to: [74002684](https://github.com/dotnet/docfx/pull/583#discussion_r74002684) [](ancestors = 74002684)